lib/connect_four.rb

```diff
@@ -32,7 +32,7 @@ def play_game
         while !game_over?
```

Comment:
saw your potential refactor on lines 17-25. do you think it's a needed refactor? I think the while !game_over? is working the same as that loop would, and I wrote a new method called "restart_game" which runs in play_game once the game_over? == true